octoprint_mrbeam/printing/profile.py

```diff
@@ -263,14 +268,51 @@ def _load_default(self, defaultModel=None):
             raise InvalidProfileError()
         return profile
 
+    def _save_to_path(self, path, profile, allow_overwrite=False):
```

Comment:
Alright, now please add docstring + what changed compared to the overloaded function